packages/bitcore-payid/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  AddressDetailsType,
+  verifySignedAddress,
+} from '@payid-org/utils';
+import { JWS } from 'jose';
+import { IVerifyPayId } from './index.d';
+
+class PayId {
+  constructor() {}
+
+  /**
+   * Sign a payId address with the given identityKey
+   * @param {string} payId
```

Comment:
would be nice to expecify examples for all parameters.

Reply:
Done